examples/utils/example_client.rs

```diff
@@ -7,10 +7,9 @@
 // permissions and limitations relating to use of the SAFE Network Software.
```

Comment:
I think this whole file is removed in another commit, so happy we remove now or later as we grab those other commits.